### PR TITLE
doc: fix grouping of mtd_native

### LINKED ISSUE #6824: Wrongly placed doc modules
HD44780 and Native MTD are accidentally placed at the top level instead of under the proper group.

### PR DESCRIPTION
**Pull request by miri64**
Partly fixes #6824.

### CODE REVIEW

File: cpu/native/mtd/mtd_native.c [hunk 1]
```diff
@@ -7,7 +7,6 @@
  */
 
 /**
- * @ingroup     mtd_native
```
Comment (miri64) [inline cpu/native/mtd/mtd_native.c]: See #6830 for reasoning for removal

**Comment by vincent-d** [comment]
OK to squash

**Comment by miri64** [comment]
Squashed

**Comment by vincent-d** [comment]
Murdock2 is happy, go.

**Comment by miri64** [comment]
Murdock is still the one to primarily look out for, but for a doc fix this is acceptable.

**Pull request merged**